package.json

```diff
@@ -8,7 +8,7 @@
   "license": "Unlicense",
   "scripts": {
     "dev": "babel-node -- src/csv2invoices.js",
-    "test": "babel-node -- ./test.js",
+    "dev-run": "babel-node -- src/index.js",
```

Comment:
feel free to rewrite the `dev` script by your new one and remove the `test` as you did. update the `dev-run` in the README then as well